vision/src/autogluon/vision/predictor/predictor.py

```diff
@@ -400,6 +405,30 @@ def fit(self,
             logging.getLogger('gluoncv.auto.tasks.image_classification').propagate = False
             logging.getLogger("ImageClassificationEstimator").propagate = False
             logging.getLogger("ImageClassificationEstimator").setLevel(log_level)
+
+        # TODO: remove this check when pytorch model supports other problems
+        if self._problem_type != MULTICLASS:
```

Comment:
Does TIMM support binary classification if it is specified as multiclass with 2 classes?